package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "@leadhome/oxygen",
-  "version": "0.0.5",
+  "version": "1.0.0",
```

Comment:
Should this really be a "full" release?

Reply:
Maybe not, but the npm versioning was messed up and the minor version was already taken.

Reply:
I think it should be 0.0.7?

Reply:
Patches are for bug fixes, this should ideally increment a minor version.
But 0.1.0 already exists for some reason.

Reply:
I can't see a 0.1.0 on NPM so not sure where it's picking that up from.